docs/installation.md

```diff
@@ -75,6 +75,29 @@ Run the unit tests using jest by using:
 npm test
 ```
 
+### Rosetta API
+
+#### Prerequisites
+``
+Go 1.13+
+``
+
+To start the Rosetta API ensure you have the necessary [configuration](configuration.md) populated and run:
+
+```console
+cd hedera-mirror-rosetta
+go run cmd/*
+```
+
+#### Rosetta CLI Validation
+
+After you have started the Rosetta API, in another terminal run:
+
+```console
+cd hedera-mirror-rosetta/validation
```

Comment:
- I think that it would be better to explicitly separate the 2 commands, f.e:

1. CD to `hedera-mirror-rosetta/validation`
2. Execute `./run-validations.sh` .... (network props info)

- I think that it would be useful if we add information that if no `NETWORK` property is provided, the validation runs against the `DEMO` data and If `testnet` is provided it runs against that network

Reply:
Regarding bullet 1:
* all examples above are done in a single code block
* easier to be copied